macros/sfdc_formula_view_sql.sql

```diff
@@ -12,7 +12,7 @@
             --The select statement must explicitly query from and join from the source, not the target. The replace filters point the query to the source.
             {% if ' from ' in v %}
                 {%- set v = v | replace(' from ',' from ' + source(source_name,'fivetran_formula') | string ) -%}
-                {% if target.type == 'bigquery' %} {%- set v = v | replace('`fivetran_formula`','') -%} 
+                {% if target.type == 'bigquery', 'spark', 'databricks' %} {%- set v = v | replace('`fivetran_formula`','') -%} 
```

Comment:
I don't believe this will actually work. The `==` operator intended for only a single value check. Whereas the `in` operator allows you to check a number of different cases.

Reply:
Additionally, it seems that when merging the customer PR this same update was not applied to the code on line 22. Can you make the same update there.

Reply:
Whoops you're right-- updated